libs/shared-utils/src/lib/tailwind-cn-util.ts

```diff
@@ -0,0 +1,6 @@
+import { clsx, type ClassValue } from 'clsx'
+import { twMerge } from 'tailwind-merge'
+
+export function cn(...inputs: ClassValue[]) {
+  return twMerge(clsx(inputs))
+}
```

Comment:
Does `cn` stand for classnames? Again, I would use a clearer name here, if it's not a strict convention from tailwind.

Reply:
➕

Reply:
Tbh, I am not sure about this one. It is not a convention from Tailwind, but might be from shacn/ui. 

This function gets automatically imported and used in the component generated by shadcn/ui CLI. For example, here in `button.tsx`:
<img width="608" alt="image" src="https://github.com/user-attachments/assets/b89f4ff6-d02e-4d00-8037-ce045279e9d8">

Reply:
So, I tried it out. We can rename this function, but then we need to manually update the import statement and the function name in each generated (or copy-pasted) shadcn/ui component where it is used.

Reply:
I wouldn't bother then. Good to stick with the default.

Reply:
➕